tests/OneSpanSign.test.ts

```diff
@@ -0,0 +1,72 @@
+import { Response } from 'node-fetch';
+import { Readable } from 'stream';
+
+import { OneSpanSign, CreatePackageRequestPayload, UploadDocumentRequestPayload } from '../src';
+import { mockedFetch } from './setup/mockNodeFetch';
+
+const MOCK_API_KEY = 'demoKey';
+const MOCK_API_URL = 'http://demo.com';
+
+describe('OneSpanSign', () => {
+  let oneSpan: OneSpanSign;
+
+  beforeAll(() => {
+    mockedFetch.mockResolvedValue({ ok: true, json: () => Promise.resolve({}) } as Response);
+    oneSpan = new OneSpanSign(MOCK_API_KEY, MOCK_API_URL);
+  });
+
+  afterAll(() => {
+    mockedFetch.mockReset();
+  });
+
+  afterEach(() => {
+    mockedFetch.mockClear();
+  });
+
+  describe('createPackage', () => {
+    it('sends the provided payload as request body', async () => {
+      const payload: CreatePackageRequestPayload = {
+        name: 'mock package',
+      };
+
+      await oneSpan.createPackage(payload);
+
+      expect(mockedFetch.mock.calls[0]).toMatchSnapshot();
+    });
+  });
+
+  describe('uploadDocument', () => {
+    it.each`
+      type        | data                        | expectMatcher
+      ${'buffer'} | ${Buffer.from('abc1234')}   | ${(data: any) => data}
+      ${'stream'} | ${Readable.from('abc1234')} | ${(data: any) => expect.objectContaining({ source: data })}
+    `('sends the provided payload and $type as request body', async ({ data, expectMatcher }) => {
+      const payload: UploadDocumentRequestPayload = {
+        name: 'mock document',
+      };
+      const packageId = 'mockId';
+
+      await oneSpan.uploadDocument(packageId, payload, data);
+
+      // Had to compare the values this way instead of using a snapshot because the boundary value is different each time
```

Comment:
Just fyi, if you ever want to set a custom boundary on a FormData object you can do it this way:

```
const formData = new FormData()
formData._boundary = '--------------------------my-custom-boundary'

console.info(formData.getHeaders())  
//  { 'content-type': 'multipart/form-data; boundary=--------------------------my-custom-boundary' }
```
Of course that'd be a little weird to add a `customBoundary` parameter to your `uploadDocument` method just to be able to use snapshots.

Anyway... I thought this might be good for peeps to know.